nengo_bones/templates/pyproject.toml.template

```diff
@@ -0,0 +1,16 @@
+# Automatically generated by nengo-bones, do not edit this file directly
+
+[build-system]
+requires = ["setuptools", "wheel"]
+
+[tool.black]
+target-version = ['py35']
```

Comment:
Should we include py36/py37 here as well? I thinkkk all our nengo-bones-controlled projects support all 3?

Reply:
Yeah sounds good. I wasn't too sure how this settings works, I was mostly trying to make sure Black doesn't use a bunch of Python 3.6+ only features, which it will do if it can't automatically infer that you're using Python 3.5.